scripts/vint.py

```diff
@@ -0,0 +1,101 @@
+# Script to convert Vint's Linter warnings to valid GitHub warning or error
+#
+# Copyright (C) 2021    Lucas Ritzdorf, Luca Zeuch
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import json
+import shutil
+import subprocess
+import sys
+
+# Generate our vint command and fail if the executable is not found.
+def generate_command():
+    exe = shutil.which("vint")
+    args = [
+        exe,
+        "--json",
+        "."
+        ]
+
+    if exe == None:
+        print("No vint executable found. Install it with pip install vim-vint.")
+        sys.exit(1)
+
+    return ' '.join(args)
```

Comment:
```suggestion
    exe = shutil.which("vint")
    if not exe:
        print("No vint executable found. Install it with pip install vim-vint.")
        sys.exit(1)
    else:
       return f"{exe} --json ."
```
No need for `args`, and checking `if X` instead of explicit comparison with `None` (which in any case should use `is` for object equality) is more idiomatic.